.changeset/honest-ants-fold.md

```diff
@@ -0,0 +1,9 @@
+---
+'@farfetched/core': patch
+'@farfetched/misc': patch
+'@farfetched/react': patch
+'@farfetched/runtypes': patch
+'@farfetched/solid': patch
+---
+
+use a rollup executor to build libraries (bundles .d.ts files and allows multiple module formats)
```

Comment:
In my mind, this changelog is meaningless for users, they do not care which compiler is user. I think, it is better to describe the result of the changes, like this one


```suggestion
Add ESM build
```

What do you think?